### ISSUE TITLE
Startup warning | IllegalArgumentException: Material cannot be null

### ISSUE THREAD

**Issue by Aztlon** [StrangeOne101/VocalMimicryMenu#1]
On plugin load, I get the following warning:
```[21:43:09 WARN]: [VocalMimicryMenu] Plugin VocalMimicryMenu v1.1 generated an exception while executing task 137
java.lang.IllegalArgumentException: Material cannot be null
        at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145) ~[guava-31.1-jre.jar:?]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:97) ~[purpur-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:85) ~[purpur-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:72) ~[purpur-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:58) ~[purpur-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at com.strangeone101.vocalmimicrymenu.SoundBreakdown.fillAutoTextures(SoundBreakdown.java:334) ~[VocalMimicryMenu-1.1.jar:?]
        at com.strangeone101.vocalmimicrymenu.SoundBreakdown.fillSounds(SoundBreakdown.java:696) ~[VocalMimicryMenu-1.1.jar:?]
        at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftTask.run(CraftTask.java:101) ~[purpur-1.19.3.jar:git-Purpur-1884]
        at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[purpur-1.19.3.jar:git-Purpur-1884]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[purpur-1.19.3.jar:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

Using Purpur version 1.19.3 build 1884
PK 1.19.2
ProjectAddons 1.2.4

**Comment by StrangeOne101** [comment]
Yeah... lol

**Comment by StrangeOne101** [comment]
Fixed in bf5d1914ba1120b50e28d0bec38eab0dbc251b80
